app/views/catalog/_index_header_list_default.html.erb

```diff
@@ -0,0 +1,15 @@
+<%# header bar for doc items in index view -%>
+<div class="documentHeader row">
+  <%# main title container for doc partial view
```

Comment:
Will you add the css for this class (documentHeader) later?

Reply:
The CSS comes from Blacklight, this is just copied from there.